### DISCUSSION TITLE
Tasmota UI issue on and off

### DISCUSSION THREAD

**Discussion by nishantbimal** [arendst/Tasmota#15866] [Q&A] [answered]
Hi all hope you all doing well I m facing an issue when I pressing on it making devices off and when toggling off it making devices on 
![Screenshot_20220625-170328](https://user-images.githubusercontent.com/69680154/175771775-18968b85-c4cc-4d51-aeb6-663005b5027a.jpg)
And suggestions why it's happening

**Comment by sfromis** [comment, marked as answer]
Depends a lot on what your "device" is, and how it is hooked up. If it is a relay, the case may be that it is connected as active-high instead of active-low, which can be handled by configuring the gpio as `Relay_i` instead of just `Relay` (or vice versa).

**Reply by nishantbimal** [reply]
Thanks buddy it's worked
